lectures/l2.md

```diff
@@ -3,75 +3,78 @@ subtitle: "Lecture 2: Fairness Mechanisms"
 title: "GIAN Course on Fairness, Accuracy and Transparency in Machine Learning"
 author: "Suresh Venkatasubramanian"
 date: "Dec 13, 2016"
+bibliography: fatcs.bib
 
 ---
 
 # Introduction
 
-In the last lecture, we reviewed the use of machine learning in a number of "fairness-critical" environments and talked about definitions of nondiscrimination and fairness that have been proposed over the years. In this lecture, we'll look at *mechanisms* for achieving the desired goals: how can we build systems that will be fair and unbiased in the different ways we've defined so far. 
+In the last lecture, we reviewed the use of machine learning in a number of "fairness-critical" environments and talked about definitions of nondiscrimination and fairness that have been proposed over the years. In this lecture, we'll look at *mechanisms* for achieving the desired goals: how can we build systems that will be fair and unbiased in the different ways we've defined so far.
 
-Not surprisingly, different definitions will call for different solutions, and we'll have to discuss them separately. There are three broad classes of techniques that apply independently of the measure chosen and we'll organize our exploration around these three ideas. 
+Not surprisingly, different definitions will call for different solutions, and we'll have to discuss them separately. There are three broad classes of techniques that apply independently of the measure chosen and we'll organize our exploration around these three ideas.
 
 These three approaches are:
 
-* modifying the training data so that an *oblivious* training procedure will find a fair and accurate classifier. 
+* modifying the training data so that outcome of the classifier will be fair even if we don't modify the classifier.
 * modifying the way the classifier is trained (often by modifying the cost function)
-* looking at the classifier outcomes and modifying the algorithm based on this. 
+* looking at the classifier outcomes and modifying the algorithm based on it.
 
 # Modifying the data
 
-Modifying the training data has certain attractive properties. First of all, it's an acknowledgement that we might not always realize when our data source is biased, so running the data through a "de-biaser" might just be a useful sanity check. Secondly, it creates a clean separation of concerns: we can build a model that's as good as we want, using whatever tools we have at our disposal, without worrying about issues of bias or fairness at all. 
+Modifying the training data has certain attractive properties. First of all, it's an acknowledgement that we might not always realize when our data source is biased, so running the data through a "de-biaser" might just be a useful sanity check. Secondly, it creates a clean separation of concerns: we can build a model that's as good as we want, using whatever tools we have at our disposal, without worrying about issues of bias or fairness at all.
```

Comment:
@geomblog I think issues of some of other kind of bias creeping in because of data collection technique is under appreciated and it would be helpful if you give some examples.